media-gfx/hugin/hugin-2016.2.0.ebuild

```diff
@@ -52,23 +52,27 @@ DEPEND="${CDEPEND}
 
 REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 
+DOCS=( authors.txt README TODO )
+
 S=${WORKDIR}/${PN}-$(get_version_component_range 1-3)
 
 pkg_setup() {
-	DOCS="authors.txt README TODO"
-	mycmakeargs=(
-		-DBUILD_HSI=$(usex python ON OFF)
-		-DENABLE_LAPACK=$(usex lapack ON OFF)
-	)
 	use python && python-single-r1_pkg_setup
 }
 
 src_prepare() {
 	rm CMakeModules/{FindLAPACK,FindPkgConfig}.cmake || die
-
 	cmake-utils_src_prepare
 }
 
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_HSI=$(usex python ON OFF)
+		-DENABLE_LAPACK=$(usex lapack ON OFF)
```

Comment:
did you test whether you need these `ON`/`OFF`s? in 99% of cases, cmake doesn't need them, unless they do absolute weird crap like literal string comparisons.